### PR TITLE
feat(APIChannel): add `rtc_region`

### PR DESCRIPTION
**Pull request by advaith1**
https://github.com/discord/discord-api-docs/pull/2692

### CODE REVIEW

File: v8/payloads/channel.ts [hunk 1]
```diff
@@ -99,6 +99,12 @@ export interface APIChannel extends APIPartialChannel {
 	 * This may be `null` in events such as `GUILD_CREATE` when a message is not pinned
 	 */
 	last_pin_timestamp?: string | null;
+	/**
+	 * Voice region id for the voice or stage channel, automatic when set to null
```
Comment (vladfrangu) [inline v8/payloads/channel.ts:103]:
```suggestion
	 * Voice region id for the voice or stage channel, automatic when set to `null`
```

File: v8/payloads/guild.ts [hunk 2]
```diff
@@ -114,6 +114,7 @@ export interface APIGuild extends APIPartialGuild {
 	 * Voice region id for the guild
 	 *
 	 * See https://discord.com/developers/docs/resources/voice#voice-region-object
+	 * @deprecated No longer used, `rtc_region` on the channel is now used instead.
```
Comment (vladfrangu) [inline v8/payloads/guild.ts:117]: Is the field removed tho?

```suggestion
	 * @deprecated This field has been deprecated in favor of `rtc_region` on the channel.
```

File: v8/rest/channel.ts [hunk 3]
```diff
@@ -88,6 +88,12 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: text, news, store, voice
 	 */
 	parent_id?: Snowflake | null;
+	/**
+	 * Voice region id for the voice or stage channel, automatic when set to null
```
Comment (vladfrangu) [inline v8/rest/channel.ts:92]:
```suggestion
	 * Voice region id for the voice or stage channel, automatic when set to `null`
```

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: v8/payloads/guild.ts (hunk 2 above)
Comment (advaith1) [reply v8/payloads/guild.ts:117]: it still exists but doesn't do anything afaik

**Comment by vladfrangu** [comment]
Sorry @advaith1, this needs a rebase! 🙏

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**